src/gfx.asm

```diff
@@ -44,499 +44,434 @@ INCBIN "gfx/duel/box_messages.2bpp"
+	INCBIN "data/maps/tiles/gb/pokemon_dome_entrance.dimensions"
 	dw PokemonDomeEntrancePermissions
 	db FALSE ; cgb mode
-	INCBIN "data/maps/tiles/pokemon_dome_entrance.bin.lz"
+	INCBIN "data/maps/tiles/gb/pokemon_dome_entrance.bin.lz"
 PokemonDomeEntrancePermissions:
-	INCBIN "data/maps/permissions/pokemon_dome_entrance.bin.lz"
+	INCBIN "data/maps/permissions/cgb/pokemon_dome_entrance.bin.lz"
 
 PokemonDomeEntranceCGBTilemap::
-	db $20 ; width
-	db $12 ; height
+	INCBIN "data/maps/tiles/cgb/pokemon_dome_entrance.dimensions"
 	dw PokemonDomeEntranceCGBPermissions
 	db TRUE ; cgb mode
-	INCBIN "data/maps/tiles/pokemon_dome_entrance_cgb.bin.lz"
+	INCBIN "data/maps/tiles/cgb/pokemon_dome_entrance.bgmap.lz"
 PokemonDomeEntranceCGBPermissions:
-	INCBIN "data/maps/permissions/pokemon_dome_entrance_cgb.bin.lz"
+	INCBIN "data/maps/permissions/cgb/pokemon_dome_entrance.bin.lz"
 
 PokemonDomeTilemap::
-	db $20 ; width
-	db $20 ; height
+	INCBIN "data/maps/tiles/gb/pokemon_dome.dimensions"
 	dw PokemonDomePermissions
 	db FALSE ; cgb mode
-	INCBIN "data/maps/tiles/pokemon_dome.bin.lz"
+	INCBIN "data/maps/tiles/gb/pokemon_dome.bin.lz"
 PokemonDomePermissions:
-	INCBIN "data/maps/permissions/pokemon_dome.bin.lz"
+	INCBIN "data/maps/permissions/cgb/pokemon_dome.bin.lz"
 
 PokemonDomeCGBTilemap::
-	db $20 ; width
-	db $20 ; height
+	INCBIN "data/maps/tiles/cgb/pokemon_dome.dimensions"
 	dw PokemonDomeCGBPermissions
 	db TRUE ; cgb mode
-	INCBIN "data/maps/tiles/pokemon_dome_cgb.bin.lz"
+	INCBIN "data/maps/tiles/cgb/pokemon_dome.bgmap.lz"
 PokemonDomeCGBPermissions:
-	INCBIN "data/maps/permissions/pokemon_dome_cgb.bin.lz"
+	INCBIN "data/maps/permissions/cgb/pokemon_dome.bin.lz"
 
 HallOfHonorTilemap::
-	db $18 ; width
-	db $1a ; height
+	INCBIN "data/maps/tiles/gb/hall_of_honor.dimensions"
 	dw HallOfHonorPermissions
 	db FALSE ; cgb mode
-	INCBIN "data/maps/tiles/hall_of_honor.bin.lz"
+	INCBIN "data/maps/tiles/gb/hall_of_honor.bin.lz"
 HallOfHonorPermissions:
-	INCBIN "data/maps/permissions/hall_of_honor.bin.lz"
+	INCBIN "data/maps/permissions/cgb/hall_of_honor.bin.lz"
 
 HallOfHonorCGBTilemap::
-	db $18 ; width
-	db $1a ; height
+	INCBIN "data/maps/tiles/cgb/hall_of_honor.dimensions"
 	dw HallOfHonorCGBPermissions
 	db TRUE ; cgb mode
-	INCBIN "data/maps/tiles/hall_of_honor_cgb.bin.lz"
+	INCBIN "data/maps/tiles/cgb/hall_of_honor.bgmap.lz"
 HallOfHonorCGBPermissions:
-	INCBIN "data/maps/permissions/hall_of_honor_cgb.bin.lz"
+	INCBIN "data/maps/permissions/cgb/hall_of_honor.bin.lz"
 
 CardPopCGBTilemap::
-	db $14 ; width
-	db $12 ; height
+	INCBIN "data/maps/tiles/cgb/card_pop.dimensions"
 	dw NULL
 	db TRUE ; cgb mode
-	INCBIN "data/maps/tiles/card_pop_cgb.bin.lz"
+	INCBIN "data/maps/tiles/cgb/card_pop.bgmap.lz"
 
 CardPopTilemap::
-	db $14 ; width
-	db $12 ; height
+	INCBIN "data/maps/tiles/cgb/card_pop_noncgb.dimensions"
 	dw NULL
 	db TRUE ; cgb mode
-	INCBIN "data/maps/tiles/card_pop.bin.lz"
+	INCBIN "data/maps/tiles/cgb/card_pop_noncgb.bgmap.lz"
```

Comment:
The card_pop_noncgb files are in the cgb folder, but I assume that is because this tilemap has `db TRUE ; cgb mode`.
What do you think about moving the card_pop_noncgb files to the gb folder (and renaming them to card_pop) for the sake of symmetry?

(Do you think the `db TRUE ; cgb mode` is a mistake?)